docs/index.rst

```diff
@@ -67,38 +52,38 @@ For each module LangChain provides standard, extendable interfaces. LangChain al
 
    ./modules/models.rst
    ./modules/prompts.rst
-   ./modules/memory.md
-   ./modules/indexes.md
-   ./modules/chains.md
-   ./modules/agents.md
+   ./modules/memory.rst
```

Comment:
I seem to recall that we don't need to include the suffix. If that's the case, we might want to drop it so we can replace `rst` files with `md` files in the future without having to make changes in too many places.